<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurveyMaker</class>
 <widget class="QMainWindow" name="SurveyMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SurveyMaker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0" colspan="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Include questions for:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="OptionsLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Survey options</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="5">
     <widget class="QCheckBox" name="lastNameCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>This question is required.</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="6" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="PreviewLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QScrollBar" name="attributeScrollBar">
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 #98FB98, stop:1 #68DB68);
border: 1px solid black;
padding: 12px;</string>
      </property>
      <property name="text">
       <string>Create this survey in my Google Drive</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="attributeLabel">
      <property name="text">
       <string>Attributes</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="5">
     <widget class="QCheckBox" name="URMCheckBox">
      <property name="text">
       <string>Underrepresented minority status</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="7" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="TitleLabel">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>gruepr: SurveyMaker</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="5">
     <widget class="QCheckBox" name="firstNameCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>This question is required.</string>
      </property>
      <property name="text">
       <string>First / Preferred Name</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="3">
     <widget class="QCheckBox" name="sectionCheckBox">
      <property name="text">
       <string>Section</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="3" colspan="2">
     <widget class="QPlainTextEdit" name="sectionNamesTextEdit">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="placeholderText">
       <string>Enter section names on separate lines.</string>
      </property>
     </widget>
    </item>
    <item row="15" column="3" colspan="2">
     <layout class="QGridLayout" name="scheduleGrid">
      <item row="6" column="1">
       <widget class="QLineEdit" name="day3LineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Tuesday</string>
        </property>
        <property name="placeholderText">
         <string>Day 3 Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="daysLabel">
        <property name="text">
         <string>Days</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="day7LineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Saturday</string>
        </property>
        <property name="placeholderText">
         <string>Day 7 Name</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="day5CheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="day5LineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Thursday</string>
        </property>
        <property name="placeholderText">
         <string>Day 5 Name</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="day6LineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Friday</string>
        </property>
        <property name="placeholderText">
         <string>Day 6 Name</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="day4CheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="timeStartLabel">
        <property name="text">
         <string>Start Time</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="day7CheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QTimeEdit" name="timeStartEdit">
        <property name="dateTime">
         <datetime>
          <hour>9</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumTime">
         <time>
          <hour>23</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="minimumTime">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>h   AP</string>
        </property>
        <property name="time">
         <time>
          <hour>9</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="day4LineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Wednesday</string>
        </property>
        <property name="placeholderText">
         <string>Day 4 Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="day1LineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Sunday</string>
        </property>
        <property name="placeholderText">
         <string>Day 1 Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="timesLabel">
        <property name="text">
         <string>Times</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="timeEndLabel">
        <property name="text">
         <string>End Time</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="day2LineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Monday</string>
        </property>
        <property name="placeholderText">
         <string>Day 2 Name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="day3CheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QTimeEdit" name="timeEndEdit">
        <property name="dateTime">
         <datetime>
          <hour>17</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumTime">
         <time>
          <hour>23</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="minimumTime">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>h   AP</string>
        </property>
        <property name="time">
         <time>
          <hour>17</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="day2CheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="day6CheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="day1CheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="daysComboBox">
        <item>
         <property name="text">
          <string>All Days</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weekdays</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weekends</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom Days</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0" colspan="5">
     <widget class="QCheckBox" name="genderCheckBox">
      <property name="text">
       <string>Gender</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="5">
     <widget class="QCheckBox" name="additionalQuestionsCheckBox">
      <property name="text">
       <string>Add a placeholder for additional questions</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="5" rowspan="16">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="13" column="3" colspan="2">
     <widget class="QPlainTextEdit" name="attributeTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="placeholderText">
       <string>Enter the text of attribute question 1.</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QSpinBox" name="attributeCountSpinBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="3">
     <widget class="QCheckBox" name="scheduleCheckBox">
      <property name="text">
       <string>Schedule</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5">
     <widget class="QLineEdit" name="surveyTitleLineEdit">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="placeholderText">
       <string>Enter a Survey Title</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="5">
     <widget class="QCheckBox" name="emailCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>This question is required.</string>
      </property>
      <property name="text">
       <string>Email Address</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="6" rowspan="14">
     <widget class="QTextEdit" name="previewText">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Oxygen Mono</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Oxygen Mono&quot;;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>firstNameCheckBox</tabstop>
  <tabstop>lastNameCheckBox</tabstop>
  <tabstop>emailCheckBox</tabstop>
  <tabstop>genderCheckBox</tabstop>
  <tabstop>URMCheckBox</tabstop>
  <tabstop>scheduleCheckBox</tabstop>
  <tabstop>sectionCheckBox</tabstop>
  <tabstop>additionalQuestionsCheckBox</tabstop>
  <tabstop>attributeTextEdit</tabstop>
  <tabstop>daysComboBox</tabstop>
  <tabstop>day1CheckBox</tabstop>
  <tabstop>day2CheckBox</tabstop>
  <tabstop>day3CheckBox</tabstop>
  <tabstop>day4CheckBox</tabstop>
  <tabstop>day5CheckBox</tabstop>
  <tabstop>day6CheckBox</tabstop>
  <tabstop>day7CheckBox</tabstop>
  <tabstop>day1LineEdit</tabstop>
  <tabstop>day2LineEdit</tabstop>
  <tabstop>day3LineEdit</tabstop>
  <tabstop>day4LineEdit</tabstop>
  <tabstop>day5LineEdit</tabstop>
  <tabstop>day6LineEdit</tabstop>
  <tabstop>day7LineEdit</tabstop>
  <tabstop>timeStartEdit</tabstop>
  <tabstop>timeEndEdit</tabstop>
  <tabstop>sectionNamesTextEdit</tabstop>
  <tabstop>previewText</tabstop>
  <tabstop>surveyTitleLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
